README.mediawiki

```diff
@@ -40,9 +40,11 @@ The following, default portlets are defined by this ZenPack:
 * And more...
 
 == Permissions ==
-* There are no owner-level permissions associated with dashboard objects. Any user may edit a dashboard if they have "Change Device" permission.
-* Users with "Add DMD Objects" permission may add a dashboard.
-* User with "Delete Objects" permission may remove a dashboard.
+* All users are able to create their own dashboards to view objects that they are authorized to see. This includes users with the "ZenUser" role or even no global role.
+* All Users in user groups are able to edit, add and remove portlets and edit dashboards that are in their Group visibility level.
+* The default dashboard is readonly for users that have no "Manage DMD" permission.
+* Only Manager are able to edit default dashboard. For all other users the default dashboard is readonly.
+* All users are able to edit, add and remove portlets and edit dashboard that is in Global visibility level. Only Owner and Manager are able to remove Global dashboards.
```

Comment:
From Conversation with Michael DeSimone:
Michael DeSimone: Users should also be able to create dashboards.
Oles Baiko: what about `All users are able to edit, add and remove portlets and edit dashboard that is in Global visibility level. Only Owner and Manager are able to remove Global dashboards.`?
Michael DeSimone I think that is fine because the dashboards can be locked.